main.go

```diff
@@ -23,15 +25,76 @@ import (
 	"github.com/rakyll/statik/fs"
 )
 
+// Theme of longcat
+type Theme struct {
+	Head image.Image
+	Body image.Image
+	Tail image.Image
+}
+
 func loadImage(fs http.FileSystem, n string) (image.Image, error) {
 	f, err := fs.Open(n)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(fmt.Errorf("theme file does not open: %s", n))
```

Comment:
Thanks. please fix this part too. (Yes, I know this part never passed)

```
return nil, fmt.Errorf("theme file does not open: %s", n)
```

Reply:
I tried to fix it ( tobynet@8c8f2c4b08e9fb43da00b10038831ef209e6dea3 )

Reply:
I consider necessary for error handlings in `loadImage` too.

e.g. `-t wlongcat` (`/theme/wlongcat/` may not exist in `statik.go`)

Before:

```shell
$ go run main.go -n 2 -H -t wlongcat
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x████████]

goroutine 1 [running]:
main.main()
        ████████████████████████████████████████████longcat/main.go:151 +0x4d5
exit status 2
````

After: (little readable)

```golang
$ go run main.go -n 2 -H -t wlongcat
2019/██/██ ██████ theme file does not open: /themes/wlongcat/data01.png
exit status 1
```